![MetaR logo](MetaR/images/MetaR-logo-4-SMALL-300x111.png)

# Description 

MetaR takes advantage of Language Workbench Technology to facilitate data analysis with the R language. 
The tool is tailored for biologists with no programming experience, as well as expert bioinformaticians and statisticians. 
MetaR is not a replacement for R, it is a tool built on top of R.

# Where MetaR can help
MetaR performs statistical analyses over RNA-Seq data.
 
A typical MetaR analysis starts from a read count matrix, where each column is a sample and each row is a gene.

Major features of MetaR include:
 * data abstraction
 * data manipulation
 * data analysis 
 * graphical visualization of the results
 * integration with virtual environments (for enhanced reproducibility)
 * versioning of your projects

# Contents 
* [Introduction to MetaR](MetaR/Introduction.md)
* [MetaR Installation Guide](MetaR/Software.md)
* [MetaR User Guide (pdf)](MetaR/booklet/MetaR_booklet.pdf)
* [MetaR Demos](https://github.com/MetaR-Languages/MetaR-demos)
* [About Us](MetaR/AboutUs.md)

### Source code
MetaR is available on [GitHub](https://github.com/MetaR-Languages/MetaR).

MetaR is currently in active development: please use the GitHub issue tracker to file enhancement requests and bug fixing.

### Release Notes
See MetaR [releases](https://github.com/MetaR-Languages/MetaR/releases) on GitHub.

### License
MetaR is open-source and released under the [Apache 2.0 license](http://www.apache.org/licenses/LICENSE-2.0).

## Citation
If you use MetaR in a paper, please cite:

> Fabien Campagne, William ER Digan, Manuele Simi
> MetaR: simple, high-level languages for data analysis with the R ecosystem bioRxiv 2015
> doi: http://dx.doi.org/10.1101/030254

_MetaR is offered by the Informatics Core at the [Clinical and Translational Science Center (CTSC)](https://ctscweb.weill.cornell.edu/about-us/ctsc-programs/informatics) at Weill Cornell Medicine (WCM)._