During our training sessions at WCM, we often receive questions (either on R, on bioninformatics and on the demo).

Here're the most common:

**Q: What are read counts:**

**R:** 
Given a file with aligned sequencing reads and a list of genomic features, a common task is to count how many reads map to each feature.
A feature is here an interval (i.e., a range of positions) on a chromosome or a union of such intervals.
In the case of RNA-Seq, the features are typically genes, where each gene is considered here as the union of all its exons. One may also consider each exon as a feature, e.g., in order to check for alternative splicing.

**Q: What is a Differential Expression test:**

**R:** 
Gene expression is the process by which information from a gene is used in the synthesis of a functional gene product, which may be proteins. A gene is declared differentially expressed if an observed difference or change in read counts or expression levels between two experimental conditions is statistically significant.
Many microarray experiments are carried out to find genes which are differentially expressed between two (or more) samples of cells. Examples abound


More coming soon.... :)